### ISSUE TITLE
Big Endian

### ISSUE THREAD

**Issue by jayvdb** [eddieshan/dicomize#2]
I see the following comment, which I assume means big endian is not supported yet

> // TODO: handle big endian reading, for now assuming little endian by default.

**Comment by eddieshan** [comment]
Yes that's correct, there should not be leftover TODOs so if you see one it is yet, ahem, to do.
